chef_master/source/server_security.rst

```diff
@@ -310,6 +310,109 @@ These newer add-ons will also write all of their secrets to ``/etc/opscode/priva
 
 While the file does not contain passwords in plaintext, it is not safe to share with untrusted users. The format of the secrets file allows Chef Infra Server deployments to conform to regulations that forbid the appearance of sensitive data in plain text in configuration files; however, it does not make the file meaningfully more secure.
 
+SSL Encryption Between Chef Infra Server and External PostgreSQL
+================================================================
+
+**New in Chef Server 13.1.13:**  Chef Server 13.1.13 introduces the capability to encrypt traffic between Chef Infra Server and an external PostgreSQL server over SSL.  These instructions are not all-encompassing and assume familiarity with PostgreSQL administration, configuration, and troubleshooting.  Consult the appropriate PostgreSQL documentation as necessary, e.g. https://www.postgresql.org/docs/9.6/ssl-tcp.html.
```

Comment:
This updates the text but keeps the docs version number at 9.6.

```suggestion
**New in Chef Infra Server 13.1.13:**  Chef Infra Server 13.1.13 introduces the ability to encrypt traffic between Chef Infra Server and an external PostgreSQL configuration over SSL.  These instructions are not all-encompassing and assume some familiarity with PostgreSQL administration, configuration, and troubleshooting.  Consult the `PostgreSQL documentation <https://www.postgresql.org/docs/9.6/ssl-tcp.html`_ for more information on connecting to PostgreSQL over an SSL connection.
```

Reply:
Used most of your re-write but left the word 'server' vs 'configuration' for now because to me 'configuration' connotes the entire system of chef server and postgres boxes together where as 'server' connotes a single box.  (actually I originally used 'configuration' in the first version of this)